packages/components/src/styles/media/media.stories.ts

```diff
@@ -35,25 +39,21 @@ export const Default = {
   }
 };
 
-export const MediaSample = {
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: `
-          <figure class="%CLASSES% max-w-xl p-4">
-            <div class="sd-copyright" style="--copyright: '© 2024 Solid Design System';">
-              <img src="./placeholders/images/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover"/>
-            </div>
-            <figcaption>%SLOT%</figcaption>
-          </figure>
-      `
-      },
-      args
-    });
-  }
+/**
+ * Use the `sd-media--inverted` class when displayed on primary background.
+ */
+
+export const Inverted = {
+  render: () => html`
+    <div class="p-4 bg-primary">
+      <figure class="sd-media sd-media--inverted p-4">
+        <img
+          src="./placeholders/images/architecture.jpg"
+          alt="A generic placeholder jpg"
+          class="aspect-video object-cover"
+        />
+        <figcaption>Inverted description</figcaption>
+      </figure>
+    </div>
+  `
```

Comment:
Could we add sth like this to explain the description?

![image](https://github.com/user-attachments/assets/b7526cb9-a27e-4073-af61-5f9db4cda640)

Reply:
figcaption is not a slot but an html element. Updated to: 

>  Use the html `figcaption` element to add a description.

What do you think about?

Reply:
I’ve updated it like that here:

![image](https://github.com/user-attachments/assets/6a9abf33-efc2-4024-aba4-66b85ffca298)

Ok?

Reply:
Perfect